core/types/transaction.go

```diff
@@ -233,6 +233,15 @@ func (tx *Transaction) IsAA() bool {
 	return tx.data.R.Sign() == 0 && tx.data.S.Sign() == 0
 }
 
+// Sponsor returns the address of the account paying for the transaction.
+func (tx *Transaction) Sponsor(s Signer) (common.Address, error) {
```

Comment:
hmm, I should probably update https://github.com/quilt/go-ethereum/pull/13/files#diff-8fecce9bb4c486ebc22226cf681416e2R1698 to use this

Reply:
agree :-)